src/client/app/containers/editor.ts

```diff
@@ -39,6 +39,16 @@ export class Editor implements AfterViewInit {
      */
     async ngAfterViewInit() {
         this._monacoEditor = await this._monaco.create(this._editor, { theme: 'vs' });
+        let editor = this._monacoEditor;
```

Comment:
I need to access it in the lambda function in addAction's run. Inside the function defintion "this" becomes "undefined"